ui/src/setupProxy.js

```diff
@@ -0,0 +1,11 @@
+const proxy = require('http-proxy-middleware');
+
+const localProxy = proxy({
+  target: 'http://localhost:8000',
```

Comment:
now we can remove the `proxy` from `package.json` and change the docs (if any) that is referring to `proxy` prop in `package.json` to refer to this one.